js/index.js

```diff
@@ -33,51 +35,122 @@ function renderEverything() {
 }
 
 function renderPepperoni() {
-  document.querySelectorAll('.pep').forEach(onePep => {
+  document.querySelectorAll(".pep").forEach(onePep => {
     if (state.pepperoni) {
-      onePep.style.visibility = 'visible';
+      onePep.style.visibility = "visible";
+      document.querySelector(".btn-pepperoni").classList.remove("active");
```

Comment:
this is a nice approach, but you also have to take in account that the logic for removing and adding the buttons should be written in the `renderButtons` function.